packages/jest-expect/src/types.ts

```diff
@@ -16,6 +16,7 @@ export type JestExpect = {
   <T = unknown>(actual: T): JestMatchers<void, T> &
     Inverse<JestMatchers<void, T>> &
     PromiseMatchers<T>;
+  addSnapshotSerializer: typeof addSerializer;
```

Comment:
Adds duplication, but it helps to work around the problem. Type is aded here to be visible for the user, but also left in augmentation to avoid casting in code. I will follow the issue in `api-extractor` and will clean up after it will be solved.

Reply:
add a comment above it?

Reply:
above this line, that is